core/dbt/contracts/graph/parsed.py

```diff
@@ -725,6 +722,7 @@ def same_url(self, old: 'ParsedExposure') -> bool:
 
     def same_contents(self, old: Optional['ParsedExposure']) -> bool:
         # existing when it didn't before is a change!
+        # metadata/tags changes are not "changes"
```

Comment:
👍 Agree with this.

Someday we may want to revisit the logic here (should a change to `depends_on` count?), but in a world where users run `-m state:modified+` in CI, I don't think an exposure marked modified would even alter behavior there.